From 084a25970e33f3223862dcc8c9132389961b7d10 Mon Sep 17 00:00:00 2001
From: Debasish Biswas <debasishbsws.dev@gmail.com>
Date: Wed, 26 Mar 2025 19:50:40 +0530
Subject: [PATCH] fix(requirements): Remove pinned Python package versions for
 Wolfi compatibility

 Removed version constraints from Python dependencies across all relevant requirements.txt files.
 In Wolfi, we use the latest available versions of Python packages from the repository.
 Pinning versions caused compatibility issues and failed builds when versions mismatched.
 This change ensures the builds succeed by using the latest compatible versions available in Wolfi.

Signed-off-by: Debasish Biswas <debasishbsws.dev@gmail.com>
---
 .../medianstop/v1beta1/requirements.txt       | 10 +++++-----
 .../tfevent-metricscollector/requirements.txt | 12 +++++------
 .../hyperband/v1beta1/requirements.txt        | 18 ++++++++---------
 .../hyperopt/v1beta1/requirements.txt         | 20 +++++++++----------
 .../nas/enas/v1beta1/requirements.txt         | 10 +++++-----
 .../optuna/v1beta1/requirements.txt           |  8 ++++----
 cmd/suggestion/skopt/v1beta1/requirements.txt | 20 +++++++++----------
 7 files changed, 49 insertions(+), 49 deletions(-)

diff --git a/cmd/earlystopping/medianstop/v1beta1/requirements.txt b/cmd/earlystopping/medianstop/v1beta1/requirements.txt
index 2e90a564..50f11bd5 100644
--- a/cmd/earlystopping/medianstop/v1beta1/requirements.txt
+++ b/cmd/earlystopping/medianstop/v1beta1/requirements.txt
@@ -1,5 +1,5 @@
-grpcio>=1.64.1
-protobuf>=4.21.12,<5
-googleapis-common-protos==1.6.0
-kubernetes==22.6.0
-cython>=0.29.24
+grpcio
+protobuf
+googleapis-common-protos
+kubernetes
+cython
diff --git a/cmd/metricscollector/v1beta1/tfevent-metricscollector/requirements.txt b/cmd/metricscollector/v1beta1/tfevent-metricscollector/requirements.txt
index 03b74c6c..05693bec 100644
--- a/cmd/metricscollector/v1beta1/tfevent-metricscollector/requirements.txt
+++ b/cmd/metricscollector/v1beta1/tfevent-metricscollector/requirements.txt
@@ -1,6 +1,6 @@
-psutil==5.9.4
-rfc3339>=6.2
-grpcio>=1.64.1
-googleapis-common-protos==1.6.0
-tensorflow==2.16.1
-protobuf>=4.21.12,<5
+psutil
+rfc3339
+grpcio
+googleapis-common-protos
+tensorflow
+protobuf
diff --git a/cmd/suggestion/hyperband/v1beta1/requirements.txt b/cmd/suggestion/hyperband/v1beta1/requirements.txt
index 47641936..d01dffbc 100644
--- a/cmd/suggestion/hyperband/v1beta1/requirements.txt
+++ b/cmd/suggestion/hyperband/v1beta1/requirements.txt
@@ -1,9 +1,9 @@
-grpcio>=1.64.1
-cloudpickle==0.5.6
-numpy>=1.25.2
-scikit-learn>=0.24.0
-scipy>=1.5.4
-forestci==0.3
-protobuf>=4.21.12,<5
-googleapis-common-protos==1.6.0
-cython>=0.29.24
+grpcio
+cloudpickle
+numpy
+scikit-learn
+scipy
+forestci
+protobuf
+googleapis-common-protos
+cython
diff --git a/cmd/suggestion/hyperopt/v1beta1/requirements.txt b/cmd/suggestion/hyperopt/v1beta1/requirements.txt
index b97d0e8d..866297f0 100644
--- a/cmd/suggestion/hyperopt/v1beta1/requirements.txt
+++ b/cmd/suggestion/hyperopt/v1beta1/requirements.txt
@@ -1,10 +1,10 @@
-grpcio>=1.64.1
-cloudpickle==0.5.6
-numpy>=1.25.2
-scikit-learn>=0.24.0
-scipy>=1.5.4
-forestci==0.3
-protobuf>=4.21.12,<5
-googleapis-common-protos==1.6.0
-hyperopt==0.2.5
-cython>=0.29.24
+grpcio
+cloudpickle
+numpy
+scikit-learn
+scipy
+forestci
+protobuf
+googleapis-common-protos
+hyperopt
+cython
diff --git a/cmd/suggestion/nas/enas/v1beta1/requirements.txt b/cmd/suggestion/nas/enas/v1beta1/requirements.txt
index 83030ce2..b62c4888 100644
--- a/cmd/suggestion/nas/enas/v1beta1/requirements.txt
+++ b/cmd/suggestion/nas/enas/v1beta1/requirements.txt
@@ -1,5 +1,5 @@
-grpcio>=1.64.1
-googleapis-common-protos==1.6.0
-cython>=0.29.24
-tensorflow==2.16.1
-protobuf>=4.21.12,<5
+grpcio
+googleapis-common-protos
+cython
+tensorflow
+protobuf
diff --git a/cmd/suggestion/optuna/v1beta1/requirements.txt b/cmd/suggestion/optuna/v1beta1/requirements.txt
index 26ac185a..11b0cf80 100644
--- a/cmd/suggestion/optuna/v1beta1/requirements.txt
+++ b/cmd/suggestion/optuna/v1beta1/requirements.txt
@@ -1,4 +1,4 @@
-grpcio>=1.64.1
-protobuf>=4.21.12,<5
-googleapis-common-protos==1.53.0
-optuna==3.3.0
+grpcio
+protobuf
+googleapis-common-protos
+optuna
diff --git a/cmd/suggestion/skopt/v1beta1/requirements.txt b/cmd/suggestion/skopt/v1beta1/requirements.txt
index 442ac490..f5cb98b6 100644
--- a/cmd/suggestion/skopt/v1beta1/requirements.txt
+++ b/cmd/suggestion/skopt/v1beta1/requirements.txt
@@ -1,13 +1,13 @@
-grpcio>=1.64.1
-cloudpickle==0.5.6
+grpcio
+cloudpickle
 # This is a workaround to avoid the following error.
 # AttributeError: module 'numpy' has no attribute 'int'
 # See more: https://github.com/numpy/numpy/pull/22607
-numpy==1.23.5
-scikit-learn>=0.24.0, <=1.3.0
-scipy>=1.5.4
-forestci==0.3
-protobuf>=4.21.12,<5
-googleapis-common-protos==1.6.0
-scikit-optimize>=0.9.0
-cython>=0.29.24
+numpy
+scikit-learn
+scipy
+forestci
+protobuf
+googleapis-common-protos
+scikit-optimize
+cython
-- 
2.39.5 (Apple Git-154)

